index.js

```diff
@@ -102,7 +102,19 @@ var parseRSS1 = function(xmlObj, callback) {
   callback("RSS 1.0 parsing not yet implemented.")
 }
 
-var parseRSS2 = function(xmlObj, callback) {
+var parseRSS2 = function(xmlObj, settings, callback) {
+
+  if (!callback) {
+    callback = settings;
+    settings = {};
+  }
+
+  if(settings && typeof settings.customFields!=='undefined')
+      ITEM_FIELDS = ITEM_FIELDS.concat(settings.customFields.items);
+
+  if(settings && typeof settings.customFields!=='undefined')
+      TOP_FIELDS = TOP_FIELDS.concat(settings.customFields.feed);
```

Comment:
We shouldn't modify `TOP_FIELDS` or `ITEM_FIELDS`. As it stands, these arrays will grow each time `parseRSS2()` is called - if it gets called a bunch of times with `customFields`, we'll get an out-of-memory error.

Instead, let's do this:
```
options.customFields = options.customFields || {};
var itemFields = ITEM_FIELDS.concat(options.customFields.item || []);
var topFields = TOP_FIELDS.concat(options.customFields.feed || []);
```

Then iterate over `itemFields`/`topFields` instead of `ITEM_FIELDS`/`TOP_FIELDS` below.

Optional: rename `TOP_FIELDS` to `FEED_FIELDS` so it's consistent with `options`.